packages/file-asset-apis/test/base/chunked-file-reader-spec.ts

```diff
@@ -68,7 +68,7 @@ describe('The chunked file reader', () => {
             offset: 100, length: 5, total: 30, path: '/test/file'
         };
         const incData = [
-            '\n{"test4": "data"}\n{"test5": data}\n{"test6": "data"}\n',
```

Comment:
Was there a good reason this was removed?

Reply:
It was testing the wrong thing, it should have been json, not ldjson